### PR TITLE
changes made as per new table

### PR DESCRIPTION
**Pull request by jagdeepbaidwan**

### CODE REVIEW

File: src/employee_management.c [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #define MIN_YEAR 2010
 #define MAX_MONTH 12
 #define MIN_MONTH 1
+#define sal_year 2020
```
Comment (Arshpreet-Singh38) [inline src/employee_management.c]: Can we consider using the system year instead ?

**Review by puneetsingh3-sysc — APPROVED** [review]
Its look good now.

File: src/employee_management.c (hunk 1 above)
Comment (jagdeepbaidwan) [reply src/employee_management.c]: That's a good idea, thanks for pointing out.

**Review by varinderdhanda — APPROVED** [review]
your code is good

**Pull request merged**